frontend/src/components/data/PatientResultsTableHeaders.js

```diff
@@ -24,5 +24,8 @@ export const patientSearchHeaderData = [
   {
     key: "nationalId",
     header: <FormattedMessage id="patient.natioanalid" />,
+  },  {
```

Comment:
This header shouldb only be displayed if the CR serach is enabled  to distinguish between the dtata sources otherwise it shouldnt display by default